qb/select.go

```diff
@@ -0,0 +1,156 @@
+package qb
+
+// SELECT reference:
+// http://docs.datastax.com/en/dse/5.1/cql/cql/cql_reference/cql_commands/cqlSelect.html
```

Comment:
Upstream documentation is at:

https://cassandra.apache.org/doc/latest/cql/dml.html#select